src/libs/actions/Report.js

```diff
@@ -576,30 +685,18 @@ function unsubscribeFromReportChannel(reportID) {
 
 /**
  * Get the report ID for a chat report for a specific
- * set of participants and redirect to it.
+ * set of participants and navigate to it.
  *
  * @param {String[]} participants
  */
-function fetchOrCreateChatReport(participants) {
-    if (participants.length < 2) {
-        throw new Error('fetchOrCreateChatReport() must have at least two participants');
-    }
+function fetchOrCreateChatReportAndNavigate(participants) {
```

Comment:
I think the impact of this change can be significantly lessened by just adding a parameter indicating we don't want to navigate. Can we try that instead?

```js
function fetchOrCreateChatReport(participants, shouldNavigate = true) {
...
}
```

Then just pass false wherever you don't need to navigate ? Or vice-a-versa..